lib/doc/worksheet.js

```diff
@@ -109,8 +110,12 @@ class Worksheet {
     // for images, etc
     this._media = [];
 
+    // worksheet protection
+    this.sheetProtection = {};
```

Comment:
Since sheet protection is an all-or-nothing feature - it would be better to initialise this to null. This would make the rendering and parsing easier.